types/node/globals.d.ts

```diff
@@ -241,9 +241,10 @@ interface Buffer extends Uint8Array {
     write(string: string, offset: number, length: number, encoding?: BufferEncoding): number;
     toString(encoding?: string, start?: number, end?: number): string;
     toJSON(): { type: 'Buffer', data: number[] };
-    equals(otherBuffer: Uint8Array): boolean;
-    compare(otherBuffer: Uint8Array, targetStart?: number, targetEnd?: number, sourceStart?: number, sourceEnd?: number): number;
-    copy(targetBuffer: Uint8Array, targetStart?: number, sourceStart?: number, sourceEnd?: number): number;
+    equals(otherBuffer: Buffer | Uint8Array): boolean;
+    compare(otherBuffer: Buffer | Uint8Array, targetStart?: number, targetEnd?: number, sourceStart?: number, sourceEnd?: number): number;
+    concat(list: Buffer[] | Uint8Array[], totalLength?: number): Buffer;
```

Comment:
`concat` is a class method and not a instance method therefore its part of the constructor and should be removed here.